src/main/java/com/hubspot/httpql/impl/UriParamParser.java

```diff
@@ -63,28 +58,24 @@ public ParsedUriParams parseUriParams(Multimap<String, String> uriParams) {
   }
 
   @SuppressWarnings("rawtypes")
-  public ParsedUriParams parseUriParams(MultivaluedMap<String, String> uriParams) {
+  public ParsedUriParams parseUriParams(Map<String, List<String>> uriParams) {
 
     final ParsedUriParams result = new ParsedUriParams();
 
     // make a copy so we can modify it
-    MultivaluedMap<String, String> params = new MultivaluedMapImpl();
-    for (Map.Entry<String, List<String>> entry : uriParams.entrySet()) {
-      if (!ignoredParams.contains(entry.getKey().toLowerCase())) {
-        params.put(entry.getKey(), entry.getValue());
-      }
```

Comment:
Is this filtering based on ignoredParams no longer needed?

Reply:
Fixed